### DISCUSSION TITLE
Minimum versions: .NET Framework and .NET Core

### DISCUSSION THREAD

**Discussion by bradwilson** [xunit/xunit#2153] [Feedback Request]
I posted two Twitter polls, and here are the results/conclusions. Thanks to everybody who voted!

## .NET Framework

<img alt=".NET Framework Poll Results" src="https://user-images.githubusercontent.com/16944/93400484-b9f43a80-f834-11ea-86eb-634df452e331.png" width="384" />

I'm not surprised 4.7.2 beats 4.8. I am surprised how many people seem not to care about it at all.

_**Conclusion:** There will be 4.7.2 support in initial release of v3. In the medium-term future, it will be dropped entirely._

## .NET Core

<img alt=".NET Core Poll Results" src="https://user-images.githubusercontent.com/16944/93400495-c37da280-f834-11ea-9d9f-cdcabec6dd0b.png" width="381" />

I'm also not surprised that 3.1 dominates over 2.1. There is less than a year left on 2.1 support.

_**Conclusion:** There will be 2.1 support in the initial release of v2. In the short-term future, it will be pushed forward to 3.1._

Also worth noting: since .NET 5 will not be an LTS, it's unlikely that we would push forward with that as a minimum; it's more likely that we would push forward to .NET 6 instead.

Feedback welcome!

**Comment by fahadabdulaziz** [comment]
Hi,
Please keep support for .NET Framework (4.7.2) for at least a year (Until .NET 6 be released), we use WinForms but not planning to upgrade to .NET 5 due not mature migration of WinForms. and I think there a many many companies sill use it with xUnit.

A side note, I think these results not a accurate, xUnit download at nuget are 94 million let's say the users are 1 million and the voters are only 363! :-)

**Reply by bradwilson** [reply]
Obviously the vote represents people on Twitter. We need to get GitHub to do polls. 😄 

Keeping .NET Framework until at least .NET 6 makes a lot of sense.

**Comment by marco-carvalho** [comment]
@bradwilson please keep support for .NET Framework 4.8, we use WinForms but not planning to upgrade to .NET Core because of massive work it requires
